tests/utils/common.py

```diff
@@ -86,13 +88,18 @@ def load_from_settings_file(self):
             self.__dict__ = load_dict_from_json_file(custom)
 
     def get_settings_path(self):
-        return os.path.join(get_root_dir(), SETTINGS_FILE)
+        return os.path.join(get_root_dir(), self.settings_file)
 
     def register_pytest_command_line_options(self, parser):
         for key, val in object_dict_items(self):
             parser.addoption("--%s" % key, action="store", default=None)
+        parser.addoption("--settings_file", action="store", default=None)
```

Comment:
Let's just call it `settings` (shorter cmd line)

Reply:
addressed